staging/src/k8s.io/apimachinery/pkg/runtime/register.go

```diff
@@ -28,7 +28,7 @@ func (obj *TypeMeta) GroupVersionKind() schema.GroupVersionKind {
 	return schema.FromAPIVersionAndKind(obj.APIVersion, obj.Kind)
 }
 
-func (obj *Unknown) GetObjectKind() schema.ObjectKind { return &obj.TypeMeta }
```

Comment:
In fact, they are unrelated and only happen to be at the same location. Unknown embeds `runtime.TypeMeta`, and the whole point of the PR is to add `GetObjectKind` also to runtime.TypeMeta (in analogy to `metav1.TypeMeta`).